### ISSUE TITLE
Acceptance Criteria - Website1 - CX

### ISSUE THREAD

**Issue by girlx** [Christopher-Green424/websiteQAExample1#16]
# Acceptance Criteria

The website should comply with WCAG 2.1 AA ADA standards.
 - [ ] Check if passed.
 
There is a navbar and footer.
 - [ ] Check if passed.
 
All of the links on the website function as intended.
 - [ ] Check if passed.
 
The logo of the club is present and readable.
 - [x] Check if passed.

The website's footer has at least four working social media links.
 - [ ] Check if passed.

The website has a consistent theme between pages.
 - [x] Check if passed.

The website has at least 5 bordered images on it.
 - [x] Check if passed.

The website has a contact page with a form and Google Maps embed.
 - [x] Check if passed.

All resources used in the creation of the website are correctly cited and available for use.
 - [ ] Check if passed.

Important Notes:
- The blue text used for "Return to top" does not comply with WCGA 2.1 AA ADA Standards.
- The page that has a form and Google Map embed wasn't obvious that it is going to be the contact page. The domain says contact on it but on the homepage, you can't find any indication of a contact page unless you were curious and click on the "We Need Volunteers" link on the header.
- The images were cited with links to the respected photographer's portfolio but the link didn't go directly to the image used. It just go to the portfolio. If the user wants to find out if this photo do belong to this photographer, the user will have to go through the respective photographer's portfolio to find the image. The information under each image was not cited.
- Only 3 of the 4 social media icons work. The Instagram link does not work.
